content/intro-to-storybook/react/ja/simple-component.md

```diff
@@ -0,0 +1,262 @@
+---
+title: '単純なコンポーネントを作る'
+tocTitle: '単純なコンポーネント'
+description: '単純なコンポーネントを隔離された環境で作りましょう'
+commit: '3d9cd8c'
+---
+
+それでは、UI コンポーネントを[コンポーネント駆動開発](https://blog.hichroma.com/component-driven-development-ce1109d56c8e) (CDD) の手法にのっとって作ってみましょう。コンポーネント駆動開発とは、UI を最初にコンポーネントから作り始めて、最後に画面を作り上げる「ボトムアップ」の開発プロセスです。CDD を用いれば UI を作る際に直面する複雑性を軽減することができます。
```

Comment:
@chameleonhead when you're able adjust the link https://blog.hichroma.com/component-driven-development-ce1109d56c8e to point at https://www.componentdriven.org/